static/etc/system-image/writable-paths

```diff
@@ -57,6 +57,7 @@
 # cloud-init
 /etc/cloud                              auto                    persistent  transition  none
 /var/lib/cloud                          auto                    persistent  none        none
+/var/lib/waagent                        auto                    persistent  transition  none
```

Comment:
Action should be `none`, not transition

Reply:
Thanks @alfonsosanchezbeato squash merged and pushed that change as a single commit.